src/main/java/org/reaktivity/nukleus/tls/internal/TlsConfiguration.java

```diff
@@ -22,8 +22,11 @@
     public static final String HANDSHAKE_WINDOW_BYTES_PROPERTY_NAME = "nukleus.tls.handshake.window.bytes";
 
     public static final String HANDSHAKE_WINDOW_FRAMES_PROPERTY_NAME = "nukleus.tls.handshake.window.frames";
+    public static final String KEY_MANAGER_FACTORY_ALGORITHM = "nukleus.tls.KeyManagerFactory.algorithm";
```

Comment:
Let's change the constant value to `nukleus.tls.key.manager.algorithm` and update the constant name accordingly.